src/components/ButtonPanel.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable react/self-closing-comp */
+/* eslint-disable no-unused-vars */
+import React from 'react';
```

Comment:
**inline comment:** 
**With the ESLint inline comment, we are indirectly changing the rules by disabling these rules, kindly fix the codebase instead in the next milestone.**

If you are using VSCode, you can delete the inline comment and hover over the error(configure eslint checks locally). And you will see your options. Click on the more info button that will take you to a GitHub repo where it is explained why you are getting this error. I find this feature very educational. Please, try to learn and implement the necessary rules so you don't have to keep this disabling line on top of each file. I believe you can do this! 💪😇